### ISSUE TITLE
life prediction example

### ISSUE THREAD

**Issue by hpeter36** [VedAstro/VedAstro.Python#11]
Hello,

Could you please add an example to life prediction with similar chart shown on the main website or at main project github?
Or just describe the process in nutshell how to do it.

Thanks,
Peter

**Comment by sengiv** [comment]
If i understand you correctly, you want to generate life prediction SVG via python code?
If that is the case: then it should technically be already possible, since full VedAstro lib is accesible from python, you just have to find out how to access via the pythonnet clr interop.....so instead of using await as in c# you use, .Result(), otherwise code should 100% same

**Comment by sengiv** [comment]
join slack or arrange a google meet i can help you get this sorted....another similar question on youtube today, not sure if that was you......hopefully by the end of this we can have working example, where you get SVG saved to local file

**Comment by hpeter36** [comment]
Hello sengiv,

yes I was on yt too :), maybe my question was too quick, I just started analyize the source code and total new in vedic astro.

it is enough for me to give a python example which returns the necessary data to construct the chart in python or js frontend.

Thanks for your lighting fast reply :)

**Comment by rickspark4** [comment]
Hello guys,
i am facing some issue while running the project,
can i get slack link or somehow connect with you in order to understand this, i also want to get that SVG chart using python.
plz help

Regards:
Rick Spark

**Comment by sengiv** [comment]
@hpeter36  sorry to say at the time of writing there is no ready demo code for python to make life predictor related stuff (i'll start work on this soon)

**Comment by sengiv** [comment]
@rickspark4 yeah this should be very possible, till the demo code is done you can try use the API first
![image](https://github.com/VedAstro/VedAstro.Python/assets/43817262/466bfeb0-dc00-4c75-a327-503bb7421b06)

**Comment by rickspark4** [comment]
hey @sengiv, thanks for quick reply
there is another issue i am having ,  you provided python demo code in this repo which is  continiously throwing below error
"TypeError: No method matches given arguments for Person..ctor:"
could you please help me with this error ?

**Comment by hpeter36** [comment]
> @hpeter36 sorry to say at the time of writing there is no ready demo code for python to make life predictor related stuff (i'll start work on this soon)

Thanks for your effort, no problem, I am still working on another project, but curious about the python demo.

**Comment by sengiv** [comment]
@hpeter36 
>  .....data to construct the chart in python or js frontend.

for js front end just make http GET call to LifeChart API link, the settings for the chart is set in the call url, sample below :

`https://api.vedastro.org/EventsChart/SteveJobs1955/Start/00:00/24/02/1955/End/00:00/05/02/2030/+08:00/29.122/PD1,PD2,PD3,PD4,PD5,PD6,PD7/General,IshtaKashtaPhalaDegree,PlanetStrengthDegree/Ayanamsa/Raman`

**Comment by hpeter36** [comment]
@sengiv thank you, it is very nice, is there a short description, what the different options means?

If I use the python lib. the calculations will happen on my local machine, there isn't API calls behind it right?
or is there a local server under the hood?

**Comment by hpeter36** [comment]
@sengiv thanks, I will play with it and of course post a working code if achieved, thanks for the great support!

**Comment by hpeter36** [comment]
@sengiv hello again, tried some basic example, wanted to create some examples and found lot of empty classes in __init__

![image](https://github.com/VedAstro/VedAstro.Python/assets/120197094/28d80f00-8fd2-4dc7-bdf1-3155c9909103)

and error "type object 'PlanetName' has no attribute 'SUN'"

Am I missing something?

Seems like this repo not in a working state, should I host own C# API server or call vedastro site?

Thanks,
